app/src/main/java/org/cirdles/topsoil/app/TopsoilMainWindow.java

```diff
@@ -373,4 +376,27 @@ void emptyTable() {
         getCurrentTable().ifPresent(TsvTable::clear);
     }
 
+    @FXML
+    void openDocumentationLink() {
+        try {
+            if (Desktop.isDesktopSupported()) {
```

Comment:
If this returns `false`, the program does nothing (not even a log), confusing the user.